packages/api/src/getUserFromContext.ts

```diff
@@ -0,0 +1,45 @@
+import type { APIGatewayProxyEvent, Context } from 'aws-lambda'
+import fetch from 'node-fetch'
+
+export type SupportedAuthTypes = 'auth0' | 'netlify'
+
+type NetlifyClientContext = Context & {
+  clientContext: {
+    user?: any
+  }
+}
+
+// TODO: Define what an Auth0 user looks like.
+const REDWOOD_AUTH_TYPE_HEADER = 'x-redwood-auth-type'
+
+export const getUserFromContext = async ({
+  event,
+  context,
+}: {
+  event: APIGatewayProxyEvent
+  context: NetlifyClientContext
```

Comment:
If this is going to be generic across solutions, might as well create a new generic context here (even if it's just a wrapper for `NetflifyClientContext`)